### DISCUSSION TITLE
Columbia STAT 8201: Deep Generative Models - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#518] [Announcements]
# Columbia STAT 8201: Deep Generative Models - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%E8%BF%9B%E9%98%B6/STAT8201/

**Comment by MisterPANDC** [comment]
这个课程是只有slides资料吗

**Comment by SamoyedY** [comment]
请问有朋友下载了slides吗? 最近发现网页已经404了Orz

**Reply by njustwhj** [reply]
在web.archive.org网站可以找回历史网站的镜像
